src/activities/GradeCandidateEntity.js

```diff
@@ -25,6 +25,15 @@ export class GradeCandidateEntity extends Entity {
 			return entity.getLinkByRel(Rels.Grades.grade).href;
 		}
 	}
+	/**
+	 * @returns {bool} True if grade-candidate is `autoPoints` i.e. maxPoints are set automatically
+	 */
+	isAutoPoints() {
```

Comment:
I think this should go into the `GradeEntity` instead of here, so that it can be retrieved from the grade's autoPoints property, instead of from an action field. This action shouldn't have an autoPoints field.

Reply:
Was just about to mention the link between this and https://github.com/Brightspace/lms/pull/17217#discussion_r770940680

I'll try and rework things 😃